modules/issue_tracker/php/endpoints/attachment.class.inc

```diff
@@ -0,0 +1,259 @@
+<?php declare(strict_types=1);
+/**
```

Comment:
This comment block can be deleted if you want #6046

Reply:
Thanks @johnsaigle, done.